deploy/olm-catalog/ibm-block-csi-operator/1.11.2/manifests/ibm-block-csi-operator.clusterserviceversion.yaml

```diff
@@ -0,0 +1,660 @@
+                - watch
+            - apiGroups:
+                - snapshot.storage.k8s.io
+              resources:
+                - volumesnapshotcontents
+              verbs:
+                - create
+                - delete
+                - get
+                - list
+                - update
+                - watch
+                - patch
+            - apiGroups:
+                - snapshot.storage.k8s.io
+              resources:
+                - volumesnapshotcontents/status
+              verbs:
+                - update
+                - patch
+            - apiGroups:
+                - snapshot.storage.k8s.io
+              resources:
+                - volumesnapshots
+              verbs:
+                - get
+                - list
+                - update
+                - watch
+            - apiGroups:
+                - storage.k8s.io
+              resources:
+                - csidrivers
+              verbs:
+                - create
+                - delete
+                - get
+                - list
+                - watch
+            - apiGroups:
+                - storage.k8s.io
+              resources:
+                - csinodes
+              verbs:
+                - get
+                - list
+                - watch
+            - apiGroups:
+                - storage.k8s.io
+              resources:
+                - storageclasses
+              verbs:
+                - get
+                - list
+                - watch
+            - apiGroups:
+                - storage.k8s.io
+              resources:
+                - volumeattachments
+              verbs:
+                - get
+                - list
+                - patch
+                - update
+                - watch
+            - apiGroups:
+                - storage.k8s.io
+              resources:
+                - volumeattachments/status
+              verbs:
+                - patch
+
+      deployments:
+        - name: ibm-block-csi-operator
+          spec:
+            replicas: 1
+            selector:
+              matchLabels:
+                app.kubernetes.io/name: ibm-block-csi-operator
+            template:
+              metadata:
+                labels:
+                  app.kubernetes.io/name: ibm-block-csi-operator
+                  csi: ibm
+              spec:
+                serviceAccountName: ibm-block-csi-operator
+                affinity:
+                  nodeAffinity:
+                    requiredDuringSchedulingIgnoredDuringExecution:
+                      nodeSelectorTerms:
+                        - matchExpressions:
+                            - key: kubernetes.io/arch
+                              operator: In
+                              values:
+                                - amd64
+                containers:
+                  - name: ibm-block-csi-operator
+                    resources:
+                      requests:
+                        memory: 50Mi
```

Comment:
let's raise this to 100Mi, like in the community YAML

Reply:
Done.

```
AzureAD+UriZiv@837003756-0TP8:uriziv1_update_version:/c/Repos/ibm-block-csi-operator> gg memory: | grep 1.11.2
deploy/olm-catalog/ibm-block-csi-operator-community/1.11.2/manifests/ibm-block-csi-operator.v1.11.2.clusterserviceversion.yaml:641:                        memory: 100Mi
deploy/olm-catalog/ibm-block-csi-operator-community/1.11.2/manifests/ibm-block-csi-operator.v1.11.2.clusterserviceversion.yaml:644:                        memory: 500Mi
deploy/olm-catalog/ibm-block-csi-operator/1.11.2/manifests/ibm-block-csi-operator.clusterserviceversion.yaml:629:                        memory: 100Mi
deploy/olm-catalog/ibm-block-csi-operator/1.11.2/manifests/ibm-block-csi-operator.clusterserviceversion.yaml:632:                        memory: 500Mi
```